File: GO/code/sum_factorial.go
package main
import "fmt"

func main () {
	var num int
	res := 0
	fmt.Println("User, enter a number: ")
	fmt.Scan(&num)
	orig := num
	for (num > 0) {
		res += num
		num -= 1
	}
	fmt.Printf("%d --> %d\n", orig, res)
}

File: C#/code/sum_factorial.cs
namespace aviv_yunker {
	class sum_factorial {
		public static void Main (string [] args) {
			string x;
			int num, res = 0;
			System.Console.WriteLine("User, enter a number: ");
			x = System.Console.ReadLine();
			num = int.Parse(x);
			while (num > 0) {
				res += num;
				num--;
			}
			System.Console.WriteLine("{0} --> {1}", x, res);	
		}
	}
}
